cmd/testrunner.go

```diff
@@ -210,3 +216,18 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 		return nil
 	}
 }
+
+func validateDataStreamsFlag(packageRootPath string, dataStreams []string) error {
+	for _, dataStream := range dataStreams {
```

Comment:
In #244 the issue was that no data streams were specified even though the `--data-streams` flag was specified. If I'm following this code right, that issue will remain even with this code because this `for` loop would be skipped completely and a `nil` will be returned from this function.

Reply:
Actually the root cause were somewhere else. CLI can complain about missing argument for the `data-streams` flag:

```elastic-package test pipeline  -v --data-streams
Error: flag needs an argument: --data-streams```

but it can't complain in this situation:

```
elastic-package test pipeline  --data-streams -v
```

(-v is considered to be a data stream)